app/crawler/fetcher.py

```diff
@@ -0,0 +1,29 @@
+from selenium import webdriver
+import requests
+
```

Comment:
I use the Fetcher class as what other languages would call an Interface. If an object gets an instance of the Fetcher class (or descendant thereof), it knows it will always have a .fetch(url) implementation